proposals/gc/Overview.md

```diff
@@ -275,7 +278,7 @@ Structures can be compared for identity:
 ```
 (same (new $point ...) (new $point ...))  ;; false
 ```
-TODO: Could even allow heterogeneous equality (equality between operands of different type), but that might lead to some discontinueties or even prevent some potential optimizations?
+TODO: Could even allow equality comparisons between operands of different type, but that might lead to some discontinuities or even prevent some potential optimizations?
```

Comment:
Argh, this is a TODO so we shouldn't bikeshed it to death, but in this case I found that the term to add more cognitive overhead than it saved, since it immediately had to be summarize with a parenthetical anyway.